spec/ref_arch_setup/install_spec.rb

```diff
@@ -61,15 +63,14 @@
         it "returns false and outputs information about the bolt run as well as an error" do
```

Comment:
Same as above...  about why calling original, and updating "it" statement, and possible "context" statement.